friends/src/components/Form/Button.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const ButtonStyled = styled.button`
```

Comment:
You might want to move your styles below your component logic to improve readability.